src/cgame/cg_buildable.cpp

```diff
@@ -731,8 +731,8 @@ void CG_InitBuildables()
 		//icon
 		if ( ( buildableIcon = BG_Buildable( buildable )->icon ) )
 		{
-		        cg_buildables[ buildable ].buildableIcon = trap_R_RegisterShader( buildableIcon, RSF_DEFAULT );
-                }
+			cg_buildables[ buildable ].buildableIcon = trap_R_RegisterShader( buildableIcon, (RegisterShaderFlags_t) ( RSF_DEFAULT | RSF_NOMIP ) );
```

Comment:
`RSF_DEFAULT` is 0, so you don't need the OR or the cast.

Reply:
Fixed